actions/prepare_branch.go

```diff
@@ -50,18 +50,9 @@ func (ctx PrepareBranch) MkReleaseLog() error {
 	filename := fmt.Sprintf("docs/changelogs/%s.md", ctx.Version.MajorMinor())
 	branch := repos.Kubo.VersionReleaseBranch(ctx.Version)
 
-	name := os.Getenv("GITHUB_USER_NAME")
-	if name == "" {
-		name = "Kubo Releaser"
-	}
-	email := os.Getenv("GITHUB_USER_EMAIL")
-	if email == "" {
-		email = "noreply+kuboreleaser@ipfs.tech"
-	}
-	token := os.Getenv("GITHUB_TOKEN")
-	if token == "" {
-		return fmt.Errorf("GITHUB_TOKEN not set")
-	}
+	name := util.GetenvPrompt("GITHUB_USER_NAME")
+	email := util.GetenvPrompt("GITHUB_USER_EMAIL")
```

Comment:
Should / could we keep the default?
Something like

```
> GITHUB_USER_EMAIL is not set. Please enter a value (default to "noreply+kuboreleaser@ipfs.tech")
> ?
```

Reply:
I was thinking that the default might be a bit misleading. As in, if the user is using kuboreleaser, the commits should be associated with the user. If it's CI (hopefully, in the near future), then we can make sure the env var is set. Does it make sense?